src/Workspaces/Core/Portable/Recommendations/Recommender.cs

```diff
@@ -28,6 +28,19 @@ public static class Recommender
             return languageRecommender.GetRecommendedSymbolsAtPosition(document, semanticModel, position, RecommendationServiceOptions.From(options, document.Project.Language), cancellationToken).NamedSymbols;
         }
 
+        public static async Task<ImmutableArray<ISymbol>> GetRecommendedSymbolsAtPositionAsync(
```

Comment:
Should we deprecate the old one? Since it's basically a race condition there, it's not really usable.

Reply:
Sure!

Reply:
What about the other overload?

Reply:
i deperecated both.

Reply:
oh wait... did i not push?